test/test_notebook.py

```diff
@@ -100,6 +100,14 @@ def test_graph_modelling_language_using_select_speaker(save=False):
     run_notebook("agentchat_graph_modelling_language_using_select_speaker.ipynb", save=save)
 
 
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.11"),
+    reason="do not run if openai is not installed or py!=3.10",

```

Comment:
This test should be moved to run under the contrib-openai workflow and then the py condition can be removed here because it will be controlled in the workflow.